Kernel_23/include/CGAL/Projection_traits_3.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2009  GeometryFactory (France)
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: LGPL-3.0-or-later OR LicenseRef-Commercial
+//
+//
+// Author(s)     : Laurent Rineau
+
+
+#ifndef CGAL_PROJECTION_TRAITS_3_H
+#define CGAL_PROJECTION_TRAITS_3_H
+
+
+#include <CGAL/internal/Triangulation_2_filtered_projection_traits_3.h>
```

Comment:
I see https://github.com/CGAL/cgal/pull/5407#issuecomment-802726241, but I don't see why you would want to keep mentions of `Triangulation_2` in the kernel. Is this for bw-compatibility issues? Shouldn't it then just be a typedef in the triangulation package?